app/subscribe/utils.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+from flask import current_app
+
+from app.constants.user_type_auth import ANONYMOUS_USER
+from app.constants.event import USER_CREATED
+from app.models import Users, Events
+from app.db_utils import create_object
+
+def create_user(user_first,
```

Comment:
You can reuse the `create_user` method in `app/share/utils.py`. Maybe we should move the method into a more general utility file.